test/server/cards/01_SOR/units/BazeMalbusTempleGuardian.spec.ts

```diff
@@ -0,0 +1,76 @@
+describe('Baze Malbus, Temple Guardian', function() {
+    integration(function (contextRef) {
+        describe('Baze Malbus\'s ability', function() {
+            describe('when the player does not have initiative', function () {
+                beforeEach(function () {
+                    contextRef.setupTest({
+                        phase: 'action',
+                        player1: {
+                            groundArena: ['baze-malbus#temple-guardian'],
+                        },
+                        player2: {
+                            groundArena: ['wampa', 'battlefield-marine'],
+                            hasInitiative: true
+                        }
+                    });
+                });
+
+                it('should not give it sentinel', function () {
+                    const { context } = contextRef;
+                    
+                    expect(context.player2).toBeActivePlayer();
+                    context.player2.clickCard(context.wampa);
+                    context.player2.clickCard(context.p1Base);
+                    expect(context.p1Base.damage).toBe(4);
+                    expect(context.player1).toBeActivePlayer();
+                });
+
+                it('should give it sentinel if the player claims initiative', function () {
```

Comment:
So because the jasmine tests are so heavy and slow and we have finite github actions time, I'm asking people to consolidate test cases as much as they can to reduce usage. Not good test practice but I don't want to run things down before we (hopefully) migrate to vitest and things aren't hella slow.

So in this case I would probably combine this test and the above into one case, with comments marking each one